src/main/java/com/marvinformatics/hibernate/json/JsonUserType.java

```diff
@@ -68,7 +68,6 @@ public boolean isMutable() {
         return true;
     }
 
-    @Override
     public void nullSafeSet(PreparedStatement st, Object value, int index, SessionImplementor session) throws HibernateException, SQLException {
```

Comment:
if this method is no longer part of the implementation, I think it's safe to take it out.